src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -38,5 +38,6 @@ internal static partial class Libraries
         internal const string Wtsapi32 = "wtsapi32.dll";
         internal const string CompressionNative = "clrcompression.dll";
         internal const string CoreWinRT = "api-ms-win-core-winrt-l1-1-0.dll";
+        internal const string MsQuic = "msquic";
```

Comment:
This should have `.dll` suffix to match the rest.

Reply:
I'm going to post an uber comment above based on this. Want to make sure everyone is on the same page.